beacon_node/beacon_chain/src/schema_change/migration_schema_v27.rs

```diff
@@ -0,0 +1,47 @@
+use crate::BeaconChainTypes;
+use ssz::Encode;
+use std::sync::Arc;
+use store::metadata::DataColumnCustodyInfo;
+use store::metadata::DATA_COLUMN_CUSTODY_INFO_KEY;
+use store::{DBColumn, Error, HotColdDB, KeyValueStoreOp};
+use tracing::info;
+
+/// Add `DataColumnCustodyInfo` entry to v27.
+pub fn upgrade_to_v27<T: BeaconChainTypes>(
```

Comment:
You also need to update current schema version here to v27:

https://github.com/sigp/lighthouse/blob/ce1685541859aae8a0b2a85ada81742c3a16f27a/beacon_node/store/src/metadata.rs#L7